groups/instance/variables.tf

```diff
@@ -4,12 +4,24 @@
 
 variable "account_name" {
   description = "The name of the AWS account we are using"
+  default     = "shared-services"
```

Comment:
I'd remove defaults for things like this. If we force people to specify the value then they'll have to consider where the instance should be, as they should

Reply:
Implemented via commit: 252d107678658864551f07a8c8958fea666199bc